tidb-cloud/connect-securely-to-serverless-clusters.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Connect Securely to Serverless Clusters
```

Comment:
```suggestion
title: Connect Securely to Serverless Tier Clusters
```
Global comment: The official name is "Serverless Tier". Please update other uses in this PR.